src/responder/gasQueue.ts

```diff
@@ -288,20 +285,4 @@ export class GasQueue {
         clonedArray.shift();
         return new GasQueue(clonedArray, this.emptyNonce, this.replacementRate, this.maxQueueDepth);
     }
-
-    /**
-     * The items in this queue that are not in the provided queue
-     * @param queue
-     */
-    public difference(queue: GasQueue) {
-        return this.queueItems.filter(tx => !queue.queueItems.includes(tx));
-    }
-
-    /**
-     * Returns true if a queue item with this identifier exists in this queue
-     * @param identifier
-     */
-    public contains(identifier: PisaTransactionIdentifier) {
-        return this.queueItems.findIndex(i => i.request.identifier.equals(identifier)) !== -1;
-    }
 }
```

Comment:
The error message in the constructor of GasQueueItem says `"Current gas price must be greater than ideal gas price"` but it should say `"Current gas price must not be smaller than ideal gas price"`

Reply:
Done.